GliaWidgets/Sources/QueuesMonitor/QueuesMonitor.swift

```diff
@@ -0,0 +1,78 @@
+import Foundation
+import GliaCoreSDK
+import Combine
+
+class QueuesMonitor {
```

Comment:
final?

Reply:
Not yet, there will be the updates in logic so I created a separate task for unit tests to complete them in the end https://glia.atlassian.net/browse/MOB-3669

Reply:
@ykyivskyi-gl I think Egor means, this class should be marked as `final`, stating that inheritance is not expected.

Reply:
@ykyivskyi-gl sorry, my bad, I didn't point out that I mean `final class`. @igorkravchenko is correct

Reply:
Will add that in next PR, thanks!